nemo/collections/nlp/modules/common/megatron/position_embedding/rotary_position_embedding.py

```diff
@@ -112,10 +122,10 @@ def forward(self, max_seq_len, offset=0):
                 t = torch.arange(self.max_seq_len_cached, dtype=self.inv_freq.dtype).to(self.inv_freq.device)
                 freqs = torch.einsum("i , j -> i j", t, self.inv_freq)
```

Comment:
Don't we need to dynamically change scale to `max_seq_len / self. max_position_embeddings`?